Misc/NEWS.d/next/Windows/2023-08-11-18-21-38.gh-issue-89240.dtSOLG.rst

```diff
@@ -0,0 +1 @@
+Allows :mod:`multiprocessing` to create pools of greater than 62 processes.
```

Comment:
should this say "on Windows" or does the "section" this news entry is in get communicated well in the ultimate generated NEWS?  I'm used to just reading them as individual line items and not seeing the section header as that can be far away at times...

Reply:
I had to [go and look](https://docs.python.org/3.13/whatsnew/changelog.html#changelog), but it seems the sections tend to be short because we don't merge together all the prereleases. So chances are the heading will be visible at the same time as the entry is.

In general, we don't specify "Windows" in the NEWS entry when it's in that section, but it does tend to get called out in commit messages (this PR will get renamed on merge because I don't like the title anymore).